privacyidea/lib/tokens/yubicotoken.py

```diff
@@ -138,6 +138,7 @@ def check_otp(self, anOtpVal, counter=None, window=None, options=None):
 
         apiId = get_from_config("yubico.id", DEFAULT_CLIENT_ID)
         apiKey = get_from_config("yubico.secret", DEFAULT_API_KEY)
+        YUBICO_URL = get_from_config("yubico.url", DEFAULT_YUBICO_URL)
```

Comment:
call this lowercase "yubico_url". But you need to change this in the function below.